hr_employee_private_wizard/tests/test_private_wizard.py

```diff
@@ -0,0 +1,50 @@
+# © 2020 - Today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from lxml import etree
+from odoo.tests.common import SavepointCase
+
+
+class TestPrivateWizard(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.gender = "female"
+
+        cls.employee = cls.env["hr.employee"].create(
+            {"name": "My Employee", "gender": cls.gender,}
+        )
+
+        cls.wizard = cls.env["hr.employee.private.wizard"].create(
+            {"employee_id": cls.employee.id,}
+        )
+
+    def test_private_field_in_fields_view_get(self):
+        result = self.wizard.fields_view_get(view_type="form")
+        assert "gender" in result["fields"]
+
+    def test_private_field_in_fields_get(self):
+        fields = self.wizard.fields_get()
+        assert "gender" in fields
+
+    def test_private_field_in_fields_view_get_arch(self):
+        result = self.wizard.fields_view_get(view_type="form")
+        tree = result["arch"]
+        field = etree.fromstring(tree).xpath("//field[@name='gender']")[0]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using lxml.etree.fromstring to parse untrusted XML data is known to be vulnerable to XML attacks. Replace lxml.etree.fromstring with its defusedxml equivalent function.](https://app.codacy.com/manual/numigi/odoo-hr-addons/pullRequest?prid=5358386)